internal/client/arg/argument.go

```diff
@@ -19,6 +19,10 @@ func (a *Arg) IsSameCommand(arg string) bool {
 	return a.commandName == arg
 }
 
+func (a *Arg) StartWith(prefix string) bool {
```

Comment:
```suggestion
func (a *Arg) StartsWith(prefix string) bool {
```
`bool` 返すなら三人称単数型がいいと思う